indoor-map/demo.js

```diff
@@ -34,17 +34,13 @@ const labelTextPreferenceOverride = [
  *
  * @param  {H.Map} map A HERE Map instance
  */
-function addVenueToMap(map) {
+function addVenueToMap(map, platform) {
+  // Indoor Maps provider interacts with a tile layer to visualize and control the Indoor Map
+  const venuesProvider = new H.venues.Provider();
+
   // Get an instance of the Indoor Maps service using a valid apikey for Indoor Maps
   const venuesService = platform.getVenuesService({ apikey: yourApikey, hrn: indoorMapHrn }, 2);
 
```

Comment:
why was the call to `getMapInfoList` removed?

Reply:
I'm not sure about this one. The change was part of a recommendation I received from Andrii Heonia.

Reply:
Please revert this change

Reply:
Done